vignettes/datatable-intro.Rmd

```diff
@@ -479,7 +479,7 @@ ans
 
 **Keys:** Actually `keyby` does a little more than *just ordering*. It also *sets a key* after ordering by setting an `attribute` called `sorted`. 
 
-We'll learn more about `keys` in the `vignette("datatable-keys-fast-subset", package="data.table")`; for now, all you have to know is that you can use `keyby` to automatically order the result by the columns specified in `by`.
+We'll learn more about `keys` in the [*Keys and fast binary search based subset*](datatable-keys-fast-subset.html) vignette; for now, all you have to know is that you can use `keyby` to automatically order the result by the columns specified in `by`.
```

Comment:
to address off-line use, https://github.com/Rdatatable/data.table/issues/6612#issuecomment-2480545839 
can this be changed from
```
[*Keys and fast binary search based subset*](datatable-keys-fast-subset.html)
```
to
```
[`vignette("datatable-keys-fast-subset", package="data.table")`](datatable-keys-fast-subset.html)
``` 
which renders as [`vignette("datatable-keys-fast-subset", package="data.table")`](datatable-keys-fast-subset.html)

Reply:
Certainly, but what do you suggest for rephrasing the text around it? or should it stay the same? (it reads a bit odd, but it might just be me)

For e.g., in the case you highlighted above, it currently reads as '... in the [*Keys and fast binary search based subset*](datatable-keys-fast-subset.html) vignette ...' so should I change it to:
i) ' ... in [`vignette("datatable-keys-fast-subset", package="data.table")`](datatable-keys-fast-subset.html) ...' (discarding the word 'vignette' for repetition)
ii) or ' ... in the [`vignette("datatable-keys-fast-subset", package="data.table")`](datatable-keys-fast-subset.html) ...' (sounding more natural as in 'the vignette')
iii) or simply ' ... in the [`vignette("datatable-keys-fast-subset", package="data.table")`](datatable-keys-fast-subset.html) vignette ...'

Reply:
whatever is easiest is fine with me

if you have time to make it sound natural, that is great

Reply:
Thanks! I think the current version looks fine, does a good job in considering both points (let me know if you need any other changes)

Reply:
Thanks! I think the current version looks fine, or at least this type of formatting does a good job in considering both points so I'm cool with it (let me know if you need any other changes)

Reply:
Thanks! I think the current version looks fine and does a good job in considering both points (hyperlinks, off-line use) so I'm cool with it (let me know if you need any other changes)